src/bin/cql-stress-cassandra-stress/settings/command/read_write.rs

```diff
@@ -327,11 +327,16 @@ fn parse_with_handles(handles: ReadWriteParamHandles) -> ReadWriteParams {
     }
 }
 
-pub fn parse_read_write_params(cmd: &Command, payload: &mut ParsePayload) -> Result<CommandParams> {
+pub fn parse_read_write_params(
+    cmd: &Command,
+    payload: &mut ParsePayload,
+) -> Result<Option<CommandParams>> {
     let args = payload.remove(cmd.show()).unwrap();
     let (parser, handles) = prepare_parser(cmd.show());
     parser.parse(args)?;
-    Ok(CommandParams::BasicParams(parse_with_handles(handles)))
+    Ok(Some(CommandParams {
+        basic_params: parse_with_handles(handles),
+    }))
```

Comment:
This function always returns either `Err()` or `Ok(Some(...))`. Does it make sense for it to return `Result<Option<CommandParams>>` instead of `Result<CommandParams>`?